Library/Homebrew/livecheck.rb

```diff
@@ -37,7 +35,8 @@ def initialize(package_or_resource)
   #
   # @param cask_name [String] name of cask to inherit livecheck info from
   # @return [String, nil]
```

Comment:
what do you think of removing the YARD type annotations? (`yard-sorbet` is just going to use the `sig` types anyway)

Reply:
The YARD types are redundant (as mentioned) but the parameter descriptions are still useful for [documentation purposes](https://rubydoc.brew.sh/Livecheck.html). If we can keep the descriptions but omit the types, that makes sense to me (i.e., we wouldn't have to waste time keeping the YARD types in line with the Sorbet method signatures).

Reply:
Agreed that it'd be nice to try and drop these types.

Reply:
It'd be nice if we could just swap out all the YARD types for sorbet `sig`s. I filed https://github.com/AaronC81/sord/issues/167 since that's the most similar project, I should have some bandwidth to take a stab at it in the summer.

Reply:
Ideally, docs would be correctly generated from

```ruby
sig { 
  params(
    # name of cask to inherit livecheck info from
    cask_name: String
  ).returns(T.nilable(String)) 
}
```